frontend/public/module/auth.js

```diff
@@ -36,8 +37,8 @@ export const authSvc = {
   email: () => loginStateItem(email),
 
   logout: (next) => {
-    setNext(next);
-    [userID, name, email].forEach(key => {
+    setNext(next || window.SERVER_FLAGS.basePath);
+    [userID, name, email, LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY].forEach(key => {
```

Comment:
I'm trying to decide if this is what we want... If you log in as another user without this set in localStorage, you will go to project default, which might be worse. If you have access to default, you probably have access to whatever this was previously set to. It also will take you to project default when you log back in as the original user.

Reply:
Note by "go to," I don't mean directly. But when you select a namespaced resource like pods, it will take you to project default even if you go the projects list initially.